### ISSUE TITLE
Looks nice !

### ISSUE THREAD

**Issue by dexception** [vinthony/depth-distillation#1]
Do you have any plans to release code with any pretrained models to check accuracy ?

**Comment by vinthony** [comment]
Thanks for your attention.
The code cleaning is ongoing and the pre-trained model and an online/offline demo will be released soon!

**Comment by yazeedhasan97** [comment]
Hi there,
are the intention of releasing the code still in mind?
your work seems very nice and promising

**Comment by vinthony** [comment]
Hi, still thanks for your attention.

I am really busy recently for some other projects.

And I will try to release the demo and pre-trained model before October.
